### ISSUE TITLE
RinFinance2019 is missing...

### ISSUE THREAD

**Issue by kboudt** [eddelbuettel/rf2#9]
I

**Comment by eddelbuettel** [comment]
Yes, as I stepped back from being sole maintainer of the website a while back.

A replacement site is said to be in the works, but has been for some tim.   /cc @lemnica
